jenkins_job_wrecker/modules/triggers.py

```diff
@@ -256,12 +256,65 @@ def ghprbtrigger(top, parent):
             ghpr['permit-all'] = get_bool(child.text)
         elif child.tag == 'autoCloseFailedPullRequests':
             ghpr['auto-close-on-fail'] = get_bool(child.text)
+        elif child.tag == 'blackListCommitAuthor':
+            ghpr['black-list-commit-author'] = child.text
+        elif child.tag == 'blackListLabels':
+            ghpr['black-list-labels'] = child.text.strip().split('\n')
+        elif child.tag == 'blackListTargetBranches':
+            ghpr['black-list-target-branches'] = [item[0].text for item in child]
+        elif child.tag == 'displayBuildErrorsOnDownstreamBuilds':
+            ghpr['display-build-errors-on-downstream-builds'] = True
+        elif child.tag == 'excludedRegions':
+            ghpr['excluded-regions'] = child.text.strip().split('\n')
+        elif child.tag == 'includedRegions':
+            ghpr['included-regions'] = child.text.strip().split('\n')
+        elif child.tag == 'skipBuildPhrase':
+            ghpr['skip-build-phrase'] = child.text
+        elif child.tag == 'whiteListLabels':
+            ghpr['white-list-labels'] = child.text.strip().split('\n')
         elif child.tag == 'whiteListTargetBranches':
             ghpr['white-list-target-branches'] = []
             for branch in child:
                 if branch[0].text is not None:
                     ghpr['white-list-target-branches'].append(branch[0].text.strip())
         elif child.tag == 'gitHubAuthId':
             ghpr['auth-id'] = child.text
+        elif child.tag == 'extensions':
+            extensions_prefix = "org.jenkinsci.plugins.ghprb.extensions."
+            for extension in child:
+                if extension.tag == extensions_prefix+"status.GhprbSimpleStatus":
+                    for attribute in extension:
```

Comment:
I'd rename `attribute` as it suggests some reference to XML attributes of the `extension` XML element, which is misleading. Maybe `extension_child`?